docs/application/native/guides/ui/tizen-ws-shell/tzsh-quickpanel.md

```diff
@@ -0,0 +1,125 @@
+# TZSH-Quickpanel
+
+TZSH-Quickpanel is a library to control the Quickpanel service window that shows notifications and system setup widgets. You can use TZSH-Quickpanel to get various information from the Quickpanel service window.
+For example, media player application needs to close the Quickpanel window during playback of video. In this case, you can use the TZSH-Quickpanel library.
+
+## Prerequisites
+
+To use the functions and structures of the TZSH-Quickpanel API (in [mobile](../../../api/mobile/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) and [wearable](../../../api/wearable/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) applications), include the `<tzsh_quickpanel.h>` header file in your application:
+
+```
+#include <tzsh_quickpanel.h>
+```
+
+## Create TZSH-Quickpanel Handler
+After you have created the main window of your application, call `tzsh_quickpanel_create_with_type` with a native window ID to create the `tzsh_quickpanel_h` structure (in [mobile](../../../api/mobile/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html#gaaa00e8e25b43c9538ca188bc43bdb3ac) and [wearable](../../../api/wearable/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html#gaaa00e8e25b43c9538ca188bc43bdb3ac) applications).
+
+```
+#include <Elementary.h>
+
+static void
+init(const char *name)
+{
+   Evas_Object *main_win;
+
+   /* create main window for the application */
+   main_win = elm_win_util_standard_add(name, name);
+
+   /* set up main window */
+   ...
+   evas_object_show(main_win);
+
+
+   tzsh_h tzsh;
+   tzsh_quickpanel_h tzsh_qp;
+   tzsh_window tz_win;
+
+   /* Get native window ID of main window */
+   tz_win = elm_win_window_id_get(main_win);
+
+   /* Create tzsh_h structure */
+   tzsh = tzsh_create(TZSH_TOOLKIT_TYPE_EFL);
+
+   /* Create tzsh_quickpanel_h structure */
+   tzsh_qp = tzsh_quickpanel_create_with_type(tzsh, tz_win, TZSH_QUICKPANEL_TYPE_SYSTEM_DEFAULT);
+```
+
+## Show or Hide Quickpanel Window
+
+To show or hide Quickpanel window when your application's window is activated, call the following functions:
+
+```
+   if (show)
+     tzsh_quickpanel_show(tzsh_qp); /* To show quickpanel window */
+   else
+     tzsh_quickpanel_hide(tzsh_qp); /* To hide quickpanel window */
+```
+
+## Get Visibility Status of Quickpanel Window
+To know the state of current visibility of Quickpanel service window, call the `tzsh_quickpanel_visible_get` function. The Quickpanel service window may be visible or invisible on the screen.
```

Comment:
Suggestion:
Can we rephrase as:
To get the current visible state of the Quickpanel service window, call `tzsh_quickpanel_visible_get`. The Quickpanel service window will be visible or invisible depending on the visible state:

Please verify and share your opinion.

Reply:
It isn't under scope to change, but I already modified other part of this article.
So I modified as your comment.